examples/pxScene2d/src/rcvrcore/optimus.js

```diff
@@ -408,16 +482,28 @@ function Application(props) {
     _externalApp.ready.then(function() {
     }, function rejection() {
       _this.log("failed to launch Spark instance app: " + _this.id);
-      _readyReject(new Error("failed to create"));
+      _readyBaseReject(new Error("failed to create"));
+      _uiReadyReject();
```

Comment:
Do we need to add a check here to confirm _readyBaseReject and _uiReadyReject are valid before calling them?  Can they still be undefined at this point?